avalon/tools/loader/app.py

```diff
@@ -400,8 +407,8 @@ def show(debug=False, parent=None, use_context=False):
             module.window.activateWindow()     # for Windows
             module.window.refresh()
             return
-        except RuntimeError as e:
-            if not str(e).rstrip().endswith("already deleted."):
+        except RuntimeError as exc:
+            if not exc.message.rstrip().endswith("already deleted."):
```

Comment:
`Exception.message` does not seem to exist in Python 3.6

```python
>>> try:
...     raise RuntimeError("TEST")
... except RuntimeError as exc:
...     print(exc.message)
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: TEST

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
AttributeError: 'RuntimeError' object has no attribute 'message'
```

This doesn't seem Python 3 compatible. See [PEP-0352](https://www.python.org/dev/peps/pep-0352/).

> The 'message' attribute is also deprecated.